projects/eslint-plugin/rules/general/lib/rules/blank-line-declaration-usage.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+module.exports = {
+	create(context) {
+		const sourceCode = context.getSourceCode();
+
+		return {
+			VariableDeclaration(node) {
+				if (
+					node.parent.type === 'ForOfStatement' ||
+					node.parent.type === 'ForStatement'
```

Comment:
We will need to add exceptions here for any iterative statement. `for in`, `while`, etc.